tidb-cloud/csv-config-for-import-data.md

```diff
@@ -0,0 +1,106 @@
+
+- Default: `True`
+
+## Delimiter
+
+- Definition: defines the delimiter used for quoting. If `Delimiter` is empty, all fields are unquoted.
+
+- Common values:
+
+    * `'"'` quotes fields with double-quote. As shown in the above screenshot, `"Michael","male"` represents two fields. Note that there must be a `,` between the two fields. If the data is `"Michael""male"` (without `,`), the import task will fail to parse. If the data is `"Michael,male"` (with only one double-quote), it represents one field.
+    * `''` disables quoting.
+
+- Default: "
+
+## Backslash-escape
+
+- Definition: whether to parse backslash inside fields as escape characters. If `Backslash-escape` is `True`, the following sequences are recognized and converted:
+
+    | Sequence | Converted to             |
+    |----------|--------------------------|
+    | `\0`     | Null character (`U+0000`)  |
+    | `\b`     | Backspace (`U+0008`)       |
+    | `\n`     | Line feed (`U+000A`)       |
+    | `\r`     | Carriage return (`U+000D`) |
+    | `\t`     | Tab (`U+0009`)             |
+    | `\Z`     | Windows EOF (`U+001A`)     |
+
+    In all other cases (for example, `\"`), the backslash is stripped, leaving the next character (`"`) in the field. The character left has no special roles (for example, delimiters) and is just an ordinary character. Quoting does not affect whether backslash is parsed as an escape character.
+
+    Take the fields in the screenshot as an example.
+
+    - If the value is `True`, `"nick name is \"Mike\""` will be parsed as `nick name is "Mike"` and written to the target table.
+    - If the value is `False`, it will be parsed as three fields: `"nick name is \"` , `Mike\`, and `""`. But it cannot be parsed correctly because the fields are not separated from each other.
+
+    For standard CSV files, if there are double-quoted characters in a field to be recorded, you need to use two double-quotes for escaping. In this case, using `Backslash-escape = True` will result in a parsing error, while using `Backslash-escape = False` will correctly parse. A typical scenario is when the imported field contains JSON content. A standard CSV JSON field is normally stored as follows:
+
+    "{""key1"":""val1"", ""key2"": ""val2""}"
+
+    In this case, you can set `Backslash-escape = False` and the field will be correctly escaped to the database as follows:
+
+    {"key1": "val1", "key2": "val2"}
+
+    If the content of the CSV source file is saved as JSON in the following way, then consider setting `Backslash-escape = True` as follows. But this is not the standard format for CSV.
+
+    "{\"key1\": \"val1\", \"key2\":\"val2\" }"
+
+- Default: `True`
+
+## Not-null and Null
+
+- Definition: the `Not-null` setting controls whether all fields are non-nullable. If `Not-null` is `False`, the string specified by `Null` is transformed to the SQL NULL instead of a specific value.
```

Comment:
```suggestion
- Definition: the **Not-null** setting controls whether all fields are non-nullable. If **Not-null** is `False`, the string specified by **Null** is transformed to the SQL NULL instead of a specific value.
```